homeassistant/components/onewire/__init__.py

```diff
@@ -1 +1,28 @@
 """The onewire component."""
+import voluptuous as vol
+
+from .const import DOMAIN
+from .onewireproxy import OneWireProxy
+
+CONFIG_SCHEMA = vol.Schema(
+    {DOMAIN: vol.Schema({}, extra=vol.ALLOW_EXTRA)}, extra=vol.ALLOW_EXTRA
+)
+
+
+async def async_setup(hass, config):
+    """Old way of setting up OneWire integrations."""
```

Comment:
comments are inconsistent. Sometimes you write OneWire and sometimes 1-Wire.

Reply:
Updated in bf99fce